### ISSUE TITLE
Increase text limit in lists

### ISSUE THREAD

**Issue by matejdro** [matejdro/PebbleDialer-Android#12]

**Comment by mcheban** [comment]
That would be great, especially for non-ASCII names. BTW is there any way to know text length in characters not in bytes?

**Comment by matejdro** [comment]
Yes there is, but length is limited in bytes due ti limits of Pebble <> Phone communication and small amount of Pebble's memory.
